examples/headsail-bsp/src/timer_unit.rs

```diff
@@ -0,0 +1,49 @@
+/**
+ * Date: 6/5/2024
+ * Author: Andreas Stergiopoulos (andreas.stergiopoulos@tuni.fi)
+ *
+ * This is the driver for the PULP timer unit. This driver is to
+ * be used in the VIRTUAL PROTOTYPE ONLY.
+ *
+ * Documentation: https://github.com/pulp-platform/timer_unit/tree/master
+ */
```

Comment:
Excellent header comment. This makes it very clear that this is VP only.

I think the date is a bit superfluous, as git already takes care of that for us, and there will be changes to this driver later which will muddy the waters with that anyway.